webapp/components/survey/TaxonomyDetails/store/state/state.js

```diff
@@ -4,6 +4,11 @@ import * as R from 'ramda'
 import * as Validation from '@core/validation/validation'
 import * as ObjectUtils from '@core/objectUtils'
 
+export const Taxonomykeys = {
```

Comment:
not clear what those keys are. can u explain ?

Reply:
not sure about the name:

```
<Table
            module={State.Taxonomykeys.taxa}
```

Maybe this keys should be in other parts of the code.

Reply:
is it used only there? then no need for extra keys object

Reply:
just spotted that i can use a string as we did in other parts! thanks!